spec/classes/archive/commmon_spec.rb

```diff
@@ -19,12 +19,12 @@
         it { is_expected.to contain_package(p) }
       end
 
-      %w[
-        docker-compose
-        cryptography
-      ].each do |p|
-        it { is_expected.to contain_python__pip(p) }
-      end
+      # %w[
```

Comment:
These pypi packages are still present in `site/profile/data/common.yaml`, why is this block being commented out?  This looks stale as we should generally be using the `docker-compose` rpm now instead of pypi packages.